components/config/opts.rs

```diff
@@ -552,7 +552,7 @@ pub fn from_cmdline_args(mut opts: Options, args: &[String]) -> ArgumentParsingR
         "Directory root with unminified scripts",
         "",
     );
-    opts.optflag("", "minibrowser", "Open minibrowser");
+    opts.optflag("", "without_minibrowser", "Open minibrowser");
```

Comment:
can this be no-minibrowser? with a dash, and “no” for simplicity

Reply:
We can not rename it to `no-minibrowser` but we can to `no_minibrowser`. We get errors from rust-analyzer if we try to use dash. Also, I think for consistency reason it would be better to follow underscore as all other candidates do that in opts.rs

Renamed to `no_minibrowser` in [f23d5f6](https://github.com/servo/servo/pull/30338/commits/f23d5f6c550eb2ff1590bcd5a385c769762e3608)

Reply:
Very strange. What error does `rust-analyzer` give? This is just a string and there are many other options with dashes.

Reply:
I guess i just need to restart my computer to fix the rust analyzer complains. Anyway renamed to `no-minibrowser` in [c38c20f](https://github.com/servo/servo/pull/30338/commits/c38c20f394573452f8dc9cfaf687a0ea97e18852)

With consistency I meant in Struct `Opts`. But you are right, we have many options with dash in the same file,.

Reply:
only the option should use a dash, the field would have to be no_minibrowser

Reply:
(or just minibrowser if we’re inverting the result of opt_present, see my suggested changes)

Reply:
thanks for the clarification delan. I think i misunderstood earlier. keeping the `minibrowser` as it was before and as you suggested